Assignment 2.Rmd

```diff
@@ -2,16 +2,28 @@
 
 ## Part I
 Start by installing the "igraph" package. Once you have installed igraph, load the package.
+```{r}
+install.packages("igraph")
+library("igraph")
+
+```
 
 Now upload the data file "discipline-data.csv" as a data frame called "D1". Each row is a disciplinary action from a teacher to a student so the first line shows that teacher "E" sent student "21" to the principal. It also shows the gender of both the teacher and student and the student's main elective field of study ("major"") and the field that the teacher instructs in ("t.expertise").
+```{r}
+data<-read.csv("discipline-data.csv")
+D1<-as.data.frame(data)
```

Comment:
You don't need to convert `data` to a data frame if you tell the `read.csv` command to account for the first row being the variable names using the `header` option: `D1 <- read.csv("discipline-data.csv", header = TRUE)`